### DISCUSSION TITLE
Automated updates?

### DISCUSSION THREAD

**Discussion by gh0sti** [yermak/AudioBookConverter#572] [General]
Are you letting azure auto update your app several times a day? Just noticing new builds a couple times a day and I'm wondering if those are needed?

**Comment by yermak** [comment]
Hi @gh0sti,

I think this is not an issue, so, converted to discussion.
I was testing chocolaty build, unfortunately it was not as smooth as I expected.
I may need to disable GitHub release not to trigger events...

**Reply by gh0sti** [reply]
Oh ok that makes sense. So what version is considered stable then?

**Comment by yermak** [comment]
6.3.2 is the latest stable for today.
